libcaf_io/src/detail/prometheus_broker.cpp

```diff
@@ -267,17 +110,12 @@ behavior prometheus_broker::make_behavior() {
 }
 
 void prometheus_broker::scrape() {
-#ifdef HAS_PROCESS_METRICS
-  // Collect system metrics at most once per second.
+  // Scrape system metrics at most once per second.
   auto now = time(NULL);
```

Comment:
```suggestion
  auto now = time(nullptr);
```

Reply:
Actually, is `nullptr == NULL` in these cases?

Reply:
Sure, but it's preferable since `nullptr` is never convertible to integers whereas `NULL` may be defined as just `0`.

Reply:
I'm not sure in which direction you are arguing. The time [man page](https://linux.die.net/man/2/time) says "non-NULL" value. Could an implementation of time compare to 0 to check that?

Reply:
Just saying we should avoid old C macros if there's a better alternative available with `nullptr`. The function takes a `time_t*` argument, so here's no ambiguity either way.